inc/datacollector/class-user.php

```diff
@@ -238,4 +240,13 @@ private function extractBookId( $key ) {
 		return $matches[1] ?? null;
 	}
 
+	/**
+	 * Add last active date to user meta
+	 *
+	 * Hooked into: save_post
+	 */
+	public static function storeLastActiveDate() {
+		update_user_meta( get_current_user_id(), self::USER_DATE_LAST_ACTIVE, gmdate( 'Y-m-d H:i:s' ) );
+	}
```

Comment:
Is there any reason for this method to be `static`?

Reply:
I did it static because the method is not dependent on instance creation